CONTRIBUTING.rst

```diff
@@ -49,31 +52,75 @@ remember that SimPEG is maintained on a volunteer basis. If you suggest an
 
 Contributing new code
 ---------------------
 
+.. _getting started: https://docs.simpeg.xyz/content/basic/installing_for_developers.html
+
+.. _practices: https://docs.simpeg.xyz/content/basic/practices.html
+
+.. _testing: https://docs.simpeg.xyz/content/basic/practices.html#testing
+
+.. _documentation: https://docs.simpeg.xyz/content/basic/practices.html#documentation
+
+.. _code style: https://docs.simpeg.xyz/content/basic/practices.html#style
+
 If you have an idea for how to improve SimPEG, please first create an issue
-and :ref:`suggest an enhancement <suggesting_enhancements>`. We will use the
+and `suggesting_enhancements`_. We will use the
 issue as a place to discuss and make decisions on the suggestion. Once you are
 ready to take action and commit some code to SimPEG, please check out
-:ref:`Getting Started for Developers <getting_started_developers>` for
-tips on setting up a development environment and :ref:`Practices <practices>`
+`getting started`_ for
+tips on setting up a development environment and `practices`_
 for a description of the development practices we aim to follow. In particular,
 
-- :ref:`testing <testing>`
-- :ref:`documentation <documentation>`
-- :ref:`code style <style>`
+- `testing`_
+- `documentation`_
+- `code style`_
 
 are aspects we look for in all pull requests. We do code reviews on pull
 requests, with the aim of promoting best practices and ensuring that new
 contributions can be built upon by the SimPEG community.
 
+.. _pr_stages:
+
+Stages of a pull request
+------------------------
+
+When first creating a pull request, try to make your suggested changes as tightly
+scoped as possible. The fewer changes you make, the faster your branch will be
+merged!
+
+If your pull request is not ready for final review, but you still want feedback
+on your current coding process please mark it as a draft pull request. Once you
+feel the pull request is ready for final review, you can convert the draft PR to
+an open PR by selecting the ``Ready for review`` button at the bottom of the page.
+
+Once a pull request is in ``open`` status and you are ready for review, please ping
+the simpeg developers in a github comment ``@simpeg/simpeg-developers`` to request a
+review. At minimum for a PR to be eligible to merge, we look for
+
+- 100% (or as close as possible) difference testing. Meaning any new code is completely tested.
+- All tests are passing.
+- All reviewer comments (if any) have been addressed.
+- A developer approves the PR.
+
+After all these steps are satisfied, a ``@simpeg/simpeg-admin`` will merge your pull request into
+the main branch (feel free to ping one of us on github).
+
+This being said, all simpeg developers and admins are essentially volunteers
+providing there time for the benefit of the community. This does mean that
+it might take some time for us to get your PR.
+
 
 Licensing
-*********
+=============
 
 The contributed code will be licensed under SimPEG's license
```

Comment:
I would rephrase this to: «[...] will be licensed under the MIT license [...]»

The term «SimPEG's license» sounds too much like it is its own license. And what we should avoid at any cost in open-source is own-baked / custom licenses. Naming it by its name avoids confusion.